### PR TITLE
Starting PR for #1, #3 and #4

### LINKED ISSUE #1: [FEATURE] Internationalization (i18n)
Replace hardcoded texts

### LINKED ISSUE #3: [FEATURE] filter to hide tasks from first column
filter/checkbox/setting to hide tasks from first column/backlog

### LINKED ISSUE #4: [FEATURE] use task colors
use task colors in gantt diagram

### PR DESCRIPTION
**Pull request by Capataloutchoupika**
Feature BlueTeck#4 : adding css, json property and update frappe-gantt.js (line 542, see official repo : https://github.com/frappe/gantt/issues/97). CSS was generated from colors in ColorModel. To use it : add class `bar-color-{color_id_of_the_task}` to the <rect.bar> element or use the "custom_class" attribute in tasks json object.

Feature BlueTeck#3 : adding checkbox to rebuild gantt with/without first colunm. Using `localstorage (localStorage.getItem("gantt_hide_first_colum")`) to save state of the checkbox.

Feature BlueTeck#1 : adding Locale files (mainly translated for BlueTeck#3) and move html template in gantt.js to a php file. Used gtranslate API to init these files.

### CODE REVIEW

**Comment by BlueTeck** [comment]
That was fast 😃 I've invited you as collaborator to this project as well.

File: Assets/gantt.js [hunk 1]
```diff
@@ -1,112 +1,130 @@
...
+                    }
+                }
+                const start_date = format_date(task._start);
+                const end_date = format_date(task._end);
+                // TODO: add more info and translate text
+                var html = jQuery('#gantt-popup-template').html();
+                var matches = html.match(/\$\{([\w-_\.]*)\}/g);
+                if (matches) {
+                    jQuery.each(matches, function (i, taskValue) {
+                        var _taskValue = eval(taskValue.slice(2, -1));
```
Comment (BlueTeck) [inline Assets/gantt.js:26]: eval is evil 🤔 

maybe just `var _taskValue = task[taskValue.slice(2, -1).split(".")[1]];`

File: Template/task_gantt/show.php [hunk 2]
```diff
@@ -12,14 +12,19 @@
...
-            id="gantt-chart"
-            data-save-url="<?php echo $this->url->href('TaskGanttController', 'save', ['project_id' => $project['id'], 'plugin' => 'Gantt']); ?>"
-            data-records='<?php echo json_encode($tasks); ?>'
-        ></div>        
+        <div id="gantt-chart"
+             data-save-url="<?php echo $this->url->href('TaskGanttController', 'save', ['project_id' => $project['id'], 'plugin' => 'Gantt']); ?>"
+             data-records='<?php echo json_encode($tasks); ?>'
+        ></div>
+        <div id="gantt-popup-template" class="hide">
+            <?= $this->render('Gantt:task_gantt/popup', array('project_id' => $project['id'], 'tasks' => $tasks)) ?>
```
Comment (BlueTeck) [inline Template/task_gantt/show.php:26]: Should be `'project' => $project` or the hook has to use project_id

File: Assets/gantt.js (hunk 1 above)
Comment (Capataloutchoupika) [reply Assets/gantt.js]: That's right ! I corrected that in 065b6bc54c8f60f924c5087cde401ad981df169d
Now, without `eval()`, we can't use multidimensional object like `task.xx.xx`, only `task.xx`. I'll check later how can I manage that.

File: Template/task_gantt/show.php (hunk 2 above)
Comment (Capataloutchoupika) [reply Template/task_gantt/show.php]: My Bad ! It's a mistake ! :( 
However I used `$projet` var in popup.php and not `$project_id`... 🤦‍♂️
Correction : 96f55eeed68ee066b46a496111370018e3d5257d

**Pull request merged**